ush/ufsda/yamltools.py

```diff
@@ -112,6 +144,36 @@ def fv3_geom_dict(case, levs, ntiles, layout, io_layout):
     return outdict
 
 
+def fv3land_geom_dict(case, levs, ntiles, layout, io_layout):
+    # returns a dictionary matching FV3-JEDI global geometry entries
+    outdict = {
+        'fms initialization': {
+            'namelist filename': '$(fv3jedi_fix_dir)/fmsmpp.nml',
+            'field table filename': '$(fv3jedi_fix_dir)/field_table',
+        },
+        'akbk': '$(fv3jedi_fix_dir)/akbk'+str(levs-1)+'.nc4',
+        'layout': layout,
+        'io_layout': io_layout,
+        'npx': str(case+1),
+        'npy': str(case+1),
+        'npz': str(levs-1),
+        'ntiles': str(ntiles),
+        'field metadata override': '$(fv3jedi_fieldmetadata_dir)/gfs-land.yaml',
+
+        'time invariant fields': {
+            'state fields': {
+                'datetime': '$(LAND_WINDOW_BEGIN)',
+                'filetype': 'fms restart',
+                'skip coupler file': 'true',
+                'state variables': '[orog_filt]',
+                'datapath': '/scratch2/BMC/gsienkf/Clara.Draper/data_RnR/orog_files_Mike/',
+                'filename_orog': 'C96_oro_data.nc',
```

Comment:
This needs to be resolution dependent, right?

Reply:
Yes, I made the changes.